### DISCUSSION TITLE
Practica de CSS alternativa?

### DISCUSSION THREAD

**Discussion by GerardoL23** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#400] [Q&A]
Amigos tengo muchas dudas, esto empezando desde 0 en esto del desarollo front end, realice una pagina con la misma idea, aqui se las dejo [https://webhospitalprueba.000webhostapp.com/](url), me gustaria saber opiniones, lo que se me hace mas complicado es el acomodo el acomodo con flex, cuando tengo multiples divs, me siento perdido ocaciones.

**Comment by doguedogue** [comment]
Genial

![image](https://user-images.githubusercontent.com/23409026/158100945-c0feafa7-b998-4df3-a29c-27e8c0d2c745.png)

**Reply by JuanSalvador73** [reply]
De hecho, se ve bien. La práctica actual no exige que sea responsivo el sitio, aunque si es un plus si lo tiene. Te dejo este link de flex en youtube donde te dan una idea de como manejar los estilos en Flex:

https://www.youtube.com/watch?v=QFXSgGg-HWo
